src/test/regress/expected/create_null_dist_key.out

```diff
@@ -812,6 +812,51 @@ SELECT result FROM run_command_on_workers($$
  4
 (2 rows)
 
+-- foreign key to a ref table
```

Comment:
For the supported pair of table types, (from null-dist-key to reference, from null-dist-key -> null-dist-key):

1) Can we insert a few rows to see we enforce foreign key constraint? (referencing value should match a value on referenced column)

2) Can we add a few tests for ForeignKeySetsNextValColumnToDefault and EnsureSupportedFKeyOnDistKey checks.
    For EnsureSupportedFKeyOnDistKey; given that they don't have distribution column, it would be enough to add one foreign key that uses ON DELETE SET DEFAULT and one other that uses ON UPDATE CASCADE or such; and then to show that we do what the foreign key action wants us to do after DELETE / UPDATE.
    (We don't need to wait until #6773 for simple DELETE / UPDATE queries, that target a single null-shard-key table without doing a join etc)